### PR TITLE
Add example for the IngressClass usage

### PR DESCRIPTION
**Pull request by SantoDE**
### What does this PR do?

Adds an example how to use the new IngressClass with Traefik

### Motivation

Make better docs

### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### Additional Notes

### CODE REVIEW

**Comment by notsureifkevin** [comment]
LGTM :+1:

File: docs/content/providers/kubernetes-ingress.md [hunk 1]
```diff
@@ -263,8 +263,33 @@ Otherwise, Ingresses missing the annotation, having an empty value, or with the
     If the Kubernetes cluster version is 1.18+,
     the new `IngressClass` resource can be leveraged to identify Ingress objects that should be processed.
     In that case, Traefik will look for an `IngressClass` in the cluster with the controller value equal to *traefik.io/ingress-controller*. 
-    
-    Please see [this article](https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/) for more information.
+    Please see [this article](https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/) for more information or the example below.
+
+    ```yaml tab="IngressClass (YAML)"
```
Comment (ldez) [inline docs/content/providers/kubernetes-ingress.md:268]:
```suggestion
    ```yaml tab="IngressClass"
```

File: docs/content/providers/kubernetes-ingress.md [hunk 2]
```diff
@@ -263,8 +263,33 @@ Otherwise, Ingresses missing the annotation, having an empty value, or with the
...
+    apiVersion: networking.k8s.io/v1beta1
+    kind: IngressClass
+    metadata: 
+      name: traefik-lb
+    spec: 
+      controller: traefik.io/ingress-controller
+    ```
+  
+    ```yaml tab="Example Ingress (YAML)"
```
Comment (ldez) [inline docs/content/providers/kubernetes-ingress.md:277]:
```suggestion
    ```yaml tab="Ingress"
```

**Review by ldez — CHANGES_REQUESTED** [review]

**Review by ldez — APPROVED** [review]

**Review by rtribotte — APPROVED** [review]
LGTM 👌

**Review by jbdoumenjou — APPROVED** [review]
LGTM

**Pull request merged**